editor/editor_help.cpp

```diff
@@ -1690,6 +1722,10 @@ EditorHelp::EditorHelp() {
 EditorHelp::~EditorHelp() {
 }
 
+DocTools *EditorHelp::get_doc_data() {
+	_wait_for_thread();
+	return doc;
+}
 void EditorHelpBit::_go_to_help(String p_what) {
```

Comment:
Add a blank line between function declarations:

```suggestion
}

void EditorHelpBit::_go_to_help(String p_what) {
```